src/import/ht_hypertable_modify.c

```diff
@@ -0,0 +1,876 @@
+
```

Comment:
Would be good to add a comment here mentioning exactly from where and witch Postgres version this code was imported.